src/java.desktop/share/classes/javax/swing/UIDefaults.java

```diff
@@ -1142,7 +1140,6 @@ public Object createValue(final UIDefaults table) {
                         cl = ClassLoader.getSystemClassLoader();
                     }
                 }
-                ReflectUtil.checkPackageAccess(className);
                 c = Class.forName(className, true, (ClassLoader)cl);
                 SwingUtilities2.checkAccess(c.getModifiers());
```

Comment:
It looks like that it also is no longer needed(+line 1152). Maybe in a separate PR.

Reply:
Yes, this PR is just about ReflectUtils so this case will be handled in a separate PR